micro-rdk/src/common/ota.rs

```diff
@@ -241,7 +263,7 @@ impl<S: OtaMetadataStorage> OtaService<S> {
         };
 
         if self.pending_version == stored_metadata.version() {
-            log::info!("firmware is up-to-date: {}", stored_metadata.version);
+            log::info!("firmware is up-to-date: `{}`", stored_metadata.version);
```

Comment:
⬇️ - If you say "firmware is up-to-date" here, then in the log message on 270, say "firmware is out of date, proceeding with ..."